bedrock/infobar/tests.py

```diff
@@ -9,11 +9,11 @@
 from mock import patch
 
 from bedrock.mozorg.tests import TestCase
-from bedrock.tabzilla.views import template_last_modified
+from bedrock.infobar.views import template_last_modified
```

Comment:
Is there a reason to keep this infobar view around still? Afaik it was just a feature of Tabzilla, same as the transbar right?

Reply:
I'm not sure about that. I'll investigate.

Reply:
I can't find any reference to it anywhere in the code and I looked in New Relic and it gets a single hit maybe once every few days. I say we nuke it.